src/main/java/com/google/sps/data/User.java

```diff
@@ -14,20 +14,59 @@
 
 package com.google.sps.data;
 
+import java.util.ArrayList;
 import java.util.Objects;
+import java.util.List;
 
 public class User {
   private final String id;
-  private final String email;
-  private final String username;
+  private String email;
```

Comment:
Why not final anymore?

Reply:
I think, it makes sense to be able edit email or username of user.

Reply:
But won't that cause issues with identification? Usually you have to have a unique username or email and this might cause some conflicts if you invite someone named cupcake and they change their names to cake. How do you ensure that the invites are still there. Also a new user can use cupcake so they might have access to events which they are not invited to originally. I don't think it's an issue right now but have to keep in mind later.

Reply:
Yes, but these info is kept in datastore. Not in Class entity